content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx

```diff
@@ -98,12 +98,12 @@ If you do not specify a custom `output` in the `generator` block, Prisma Client
 
 ### Using a custom `output` path
 
-You can also specify a custom `output` path on the `generator` configuration, for example:
+You can also specify a custom `output` path on the `generator` configuration, for example (assuming your `schema.prisma` os located in the default `prisma` subfolder):
```

Comment:
```suggestion
You can also specify a custom `output` path on the `generator` configuration, for example (assuming your `schema.prisma` file located at the default `prisma` subfolder):
```

Reply:
```suggestion
You can also specify a custom `output` path on the `generator` configuration, for example (assuming your `schema.prisma` file is located at the default `prisma` subfolder):
```

Reply:
I feel it sounds batter without 'is', what you think ! :)

I try to read it again and again,  got confused a bit now

Reply:
We will let @mhwelander decide - it's that easy 😆 

(For me the `is` seems required in the sentence, but my English is pretty German most of the time, so who knows.)

Reply:
Sure, that will be perfect 😊